function/templates/_http-trigger.yaml

```diff
@@ -0,0 +1,9 @@
+{{/* metadata for an Azure HTTP Trigger */}}
+{{- define "function.http" }}
```

Comment:
should this be `function.prometheus` ? is there other types of http triggers you can have?

Reply:
update and done all the names